packages/pigeon/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
```

Comment:
This should be updated to 22.7.1 along with the pubspec version and the generator_tools file.

I guess this is technically a breaking change, so 23.0.0 might be more correct.